k8s/consul/Makefile

```diff
@@ -3,34 +3,53 @@ include ../gcloud.Makefile
 include ../var.Makefile
 include ../images.Makefile
 
-APP_ID ?= consul
-
-SOURCE_REGISTRY ?= marketplace.gcr.io/google
+CHART_NAME ?= consul
```

Comment:
```suggestion
CHART_NAME := consul
```

or change in mariadb-galera to `?`

Reply:
App name can be custom, but chart name should be fixed. Changed to :=